src/components/Form.js

```diff
@@ -1,19 +1,40 @@
 import React from 'react';
 import 'tachyons';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/book';
 
-const Form = () => (
-  <div className="formContainer">
-    <legend id="form-title">ADD NEW BOOK</legend>
-    <form className="d-flex">
-      <label htmlFor="title">
-        <input type="text" placeholder="Book Title" id="title" />
-      </label>
-      <label htmlFor="author">
-        <input type="text" placeholder="Author" id="author" />
-      </label>
-      <button type="submit" className="formBtn pointer">ADD BOOK</button>
-    </form>
-  </div>
-);
+const Form = () => {
+  const dispatch = useDispatch();
 
+  return (
+    <div className="formContainer">
+      <legend id="form-title">ADD NEW BOOK</legend>
+      <form className="d-flex">
+        <label htmlFor="title">
+          <input type="text" placeholder="Book Title" id="title" />
+        </label>
+        <label htmlFor="author">
+          <input type="text" placeholder="Author" id="author" />
+        </label>
+        <button
+          type="button"
+          className="formBtn pointer"
+          onClick={() => {
+            const titleInp = document.getElementById('title');
+            const authorInp = document.getElementById('author');
+            dispatch(
+              addBook({
+                title: titleInp.value,
+                author: authorInp.value,
+                id: Date.now(),
+              }),
+            );
+          }}
+        >
+          ADD BOOK
+        </button>
```

Comment:
I would have done it this exact way if it was vanilla JavaScript and the DOM and the same as you, but the way you're handling the inputs here is clearly not the React way.

- [ ] Kindly create states for the inputs fields with `useState` hook and pass each state to its input value.
- [ ] Also, please extract the function for posting the data into a function, then dispatch the data into the addBoo function, and then invoke that function either on click of the button or on submit of the form. That is the React way of doing think. Please work on the changes 👍 
- [ ] Fron the requirements, you are asked to create local states for the input fields that you dispatch 

**Check the screenshot below**
![image](https://user-images.githubusercontent.com/50941074/172659381-75da7a58-f35a-4776-8d69-69cde910d346.png)